spec/models/manageiq/providers/vmware/cloud_manager/refresher_spec.rb

```diff
@@ -40,23 +67,35 @@
     expect(described_class.ems_type).to eq(:vmware_cloud)
   end
 
-  it "will perform a full refresh" do
-    2.times do # Run twice to verify that a second run with existing data does not change anything
-      @ems.reload
-      VCR.use_cassette(described_class.name.underscore, :allow_unused_http_interactions => true) do
-        EmsRefresh.refresh(@ems)
+  ALL_REFRESH_SETTINGS.each do |settings|
+    context "with settings #{settings}" do
+      before(:each) do
+        stub_settings_merge(
+          :ems_refresh => {
+            :vmware_cloud => settings
+          }
+        )
+      end
+
+      it "will perform a full refresh" do
+        2.times do
+          @ems.reload
+          VCR.use_cassette(described_class.name.underscore, :allow_unused_http_interactions => true) do
```

Comment:
Is there a reason to allow unused http interactions? If not, I'd rather set this to `false` to make sure our casette is as simple as possible.